scripts/gui/microbe_hud.mjs

```diff
@@ -47,6 +50,108 @@ export function runMicrobeHUDSetup(){
     // Editor button is initially disabled
     document.getElementById("microbeToEditorButton").classList.add("DisabledButton");
 
+    // Compounds eompress Panel button
+    const compressPanels = document.getElementsByClassName("compressPanel");
+
+    for (const element of compressPanels) {
+        // Variables needed for changes
+        const dataToChange = {
```

Comment:
This should be created inline where this is needed. Something like:
```js
                    panelToChange.classList.add("Compress");
                    panelToChange.classList.remove("Expand");

                    // Determine different values because right now
                    // The two panels are  different in layout
                    if(panelToChange.id == "compoundsPanel") {
similar content as in 'else'
                    } else {

                        onCompressPanelClicked(panelToChange, {
                        bar: "EnvironmentBar",
                        title: "EnvironmentBarTitle",
etc...
                        });
                    }
                    
```